static/styles/portico/portico.css

```diff
@@ -2,7 +2,6 @@ body {
     background-color: hsl(0, 0%, 98%);
     font-family: "Source Sans 3", Helvetica, Arial, sans-serif;
     line-height: 150%;
-    height: 100%;
```

Comment:
The body height should be removed, as this isn't fixed now and is automatic handles by the content on the page. Having this line, the page where the scrollbar was not needed was also having one scrollbar.
Fixes this bug.